Scripts/static-analyzer/Sources/static-analyzer/CLI.swift

```diff
@@ -0,0 +1,44 @@
+import ArgumentParser
+import Foundation
+import IndexStoreDB
+import Analyzer
+
+@main
+struct MainCommand: AsyncParsableCommand {
+
+    @Option(help: "Path to the data store directory, which is typically located at '<derived-data>/Index.noindex/DataStore'")
+    var dataStorePath: String
+
+    @Option
```

Comment:
For consistency with the above?

```suggestion
    @Option(help: "Path to the .log file where you stored the xcodebuild invocation logs")
```